keps/sig-auth/3221-structured-authorization-configuration/README.md

```diff
@@ -612,11 +618,10 @@ We will have extensive unit tests during feature implementation. There would be
 for the Authorizer chain in both the old and new configuration scenarios.
 
 We will add integration tests to validate the enablement/disablement flow.
-  - When the feature is disabled, only the existing command line flag `--authorization-webhook-*`
-based mode is allowed.
-  - When the feature is enable, setting both `--authorization-config` and
-configuring an authorization webhook using the `--authorization-webhook-*`
+  - When `--authorization-config` flags is defined, the feature flag must be turned on (when feature is in Alpha).
+  - Setting `--authorization-config` along `--authorization-modes` and `--authorization-webhook-*`
 command line flags should return an error.
+  - Configuring the authorizer using legacy flags should always be allowed
```

Comment:
```suggestion
  - Configuring the authorizer using legacy flags will continue to be allowed
```

Reply:
do we need a test to ensure when cmdline flag values are updated between restarts, the corresponding values in the new structured authz config are updated?

Reply:
Since the flag → structured config translation is internal and isn't persisted, as long as that has a unit test, that's probably sufficient